test/controllers/groupings_controller_test.rb

```diff
@@ -40,6 +74,28 @@ class GroupingsControllerTest < ActionDispatch::IntegrationTest
     assert_redirected_to edit_grouping_path(@grouping2)
   end
 
+  test "#update grouping name" do
+    new_name = 'some new name'
+    put grouping_url(@grouping1), params: { grouping: { name: new_name}}
+    @grouping1.reload
+
+    assert_equal new_name, @grouping1.name
```

Comment:
Nit: You can inline reload for less LOC: `assert_equal new_name, @grouping1.reload.name`